libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -414,6 +414,8 @@ bool ofShader::setupShaderFromSource(ofShader::Source && source){
  * https://www.opengl.org/discussion_boards/showthread.php/169209-include-in-glsl?p=1192415&viewfull=1#post1192415
  */
 
+typedef std::vector<std::string> StringList;
+
 //--------------------------------------------------------------
```

Comment:
Can we remove the typedef? 
Then I think it will be good to go! 🙂

Reply:
Ups! Good catch!